packages/angular_devkit/build_angular/src/browser/schema.d.ts

```diff
@@ -0,0 +1,353 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+export interface BrowserBuilderSchema {
+  /**
+   * List of static application assets.
+   */
+  assets: AssetPattern[];
+
+  /**
+   * The name of the main entry-point file.
+   */
+  main: string;
+
+  /**
+   * The name of the polyfills file.
+   */
+  polyfills?: string;
+
+  /**
+   * The name of the TypeScript configuration file.
+   */
+  tsConfig: string;
+
+  /**
+   * Global scripts to be included in the build.
+   */
+  scripts: ExtraEntryPoint[];
+
+  /**
+   * Global styles to be included in the build.
+   */
+  styles: ExtraEntryPoint[];
+
+  /**
+   * Options to pass to style preprocessors.
+   */
+  stylePreprocessorOptions?: StylePreprocessorOptions;
+
+  /**
+   * Defines the optimization level of the build.
+   */
+  optimization: boolean;
+
+  /**
+   * Replace files with other files in the build.
+   */
+  fileReplacements: FileReplacements[];
```

Comment:
Don't think this type - 'FileReplacements' - made into the source. I've looked around for it, but haven't found it.

Reply:
@larssn it did make it in, it's something we test for in https://github.com/angular/devkit/blob/master/packages/angular_devkit/build_angular/test/browser/replacements_spec_large.ts